zipline/src/commonMain/kotlin/app/cash/zipline/internal/bridge/events.kt

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (C) 2022 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package app.cash.zipline.internal.bridge
+
+import app.cash.zipline.Call
+import app.cash.zipline.CallResult
+import app.cash.zipline.EventListener
+import app.cash.zipline.Zipline
+import app.cash.zipline.ZiplineService
+
+interface EndpointEventListener {
```

Comment:
This is the subset of EventListener that endpoint publishes to, and without a Zipline parameter. I tried wiring that in but it wrecks tests that use Endpoint but not the JS engine.